hoot-core/src/main/cpp/hoot/core/conflate/UnifyingConflator.cpp

```diff
@@ -220,7 +220,7 @@ void UnifyingConflator::apply(shared_ptr<OsmMap>& map)
   //#warning validateConflictSubset is on, this is slow.
   //_validateConflictSubset(map, _matches);
 
-  LOG_DEBUG("Post constraining match count: " << _matches.size());
+  LOG_INFO("Post constraining match count: " << _matches.size());
```

Comment:
Do we want these to remain at INFO level?  Or was this leftover from troubleshooting?